cmd/swapd/main.go

```diff
@@ -209,32 +220,8 @@ type daemon struct {
 	cancel context.CancelFunc
 }
 
-func setLogLevels(c *cli.Context) error {
-	const (
-		levelError = "error"
-		levelWarn  = "warn"
-		levelInfo  = "info"
-		levelDebug = "debug"
-	)
-
-	level := c.String(flagLog)
-	switch level {
-	case levelError, levelWarn, levelInfo, levelDebug:
-	default:
-		return fmt.Errorf("invalid log level %q", level)
-	}
-
-	_ = logging.SetLogLevel("xmrtaker", level)
-	_ = logging.SetLogLevel("xmrmaker", level)
-	_ = logging.SetLogLevel("common", level)
-	_ = logging.SetLogLevel("cmd", level)
-	_ = logging.SetLogLevel("net", level)
-	_ = logging.SetLogLevel("rpc", level)
-	return nil
-}
-
 func runDaemon(c *cli.Context) error {
-	if err := setLogLevels(c); err != nil {
+	if err := logging.SetLogLevel("*", c.String(flagLogLevel)); err != nil {
```

Comment:
This may need a partial revert.  I started looking into it, because we were not getting logs from the `monero` package. The downside of this change is that were are getting more logs that we probably want from the libp2p libraries.